src/components/cuda_pw/cuda_common.c

```diff
@@ -0,0 +1,168 @@
+#include <stdio.h>
+#include <dlfcn.h>
+#include <papi.h>
+
+#include "cuda_common.h"
+
+int get_env_PAPI_CUDA_ROOT(void)
+{
+    PAPI_CUDA_ROOT_ENV = getenv("PAPI_CUDA_ROOT");
+    if (PAPI_CUDA_ROOT_ENV == NULL) {
+        return PAPI_ECMP;
+    }
+    return PAPI_OK;
+}
+
+int load_cuda_sym(void)
+{
+    int papiErr = PAPI_OK;
```

Comment:
Don't leave commented code.

Reply:
The commented code here is for future reference. This function is not complete. We have to implement all the ways that CUDA dynamic libraries can be loaded. I found in some system libcuda.so existing inside some `stubs` directory, thus it is mentioned in a comment here.

Reply:
Comments are fine. Commented code is not.

Reply:
I have changed these to comments, not code, (i.e. uncommenting them will not result in working code). It's for future reference, and will be removed prior to release.